tests/frontend/test_engine.py

```diff
@@ -43,12 +44,14 @@ def prog():
     sf.Engine("gaussian", backend_options={"batch_size": 2, "cutoff_dim": 6}),
     sf.Engine("fock", backend_options={"batch_size": 2, "cutoff_dim": 6}),
     sf.Engine("tf", backend_options={"batch_size": 2, "cutoff_dim": 6}),
+    sf.Engine("tf", backend_options={"batch_size": 2, "cutoff_dim": 6, "dtype": tf.complex64}),
 ]
 
 engines = [
     sf.Engine("gaussian", backend_options={"cutoff_dim": 6}),
     sf.Engine("fock", backend_options={"cutoff_dim": 6}),
     sf.Engine("tf", backend_options={"cutoff_dim": 6}),
+    sf.Engine("tf", backend_options={"cutoff_dim": 6, "dtype": tf.complex128}),
```

Comment:
nice, this is the correct place to add this parametrization as far as I can tell.